klayout/drc/rule_decks/mim_b.drc

```diff
@@ -0,0 +1,149 @@
+
+  # Rule MIMTM.1: Minimum MiM bottom plate spacing to the bottom plate metal
+  ## (whether adjacent MiM or routing metal) is 1.2µm.
+  logger.info('Executing rule MIMTM.1')
+  mimtm1_l1 = topmin1_metal.separation(mimtm_virtual, transparent, 1.2.um)
+  mimtm1_l1.output('MIMTM.1', "MIMTM.1 : Minimum MiM bottom plate spacing to the bottom plate metal
+                    (whether adjacent MiM or routing metal): 1.2µm")
+  mimtm1_l1.forget
+
+  # Rule MIMTM.2: Minimum MiM bottom plate overlap of Vian-1 layer.
+  ## [This is applicable for Vian-1 within 1.06um oversize of FuseTop layer
+  ## (referenced to virtual bottom plate)] is 0.4µm.
+  logger.info('Executing rule MIMTM.2')
+  mimtm_via = top_via.overlapping(mimtm_virtual)
+  mimtm2_l1 = mimtm_via.enclosed(topmin1_metal, 0.4.um, euclidian).polygons(0.001.um)
+  mimtm2_l2 = mimtm_via.not_outside(topmin1_metal).not(topmin1_metal)
+  mimtm2_l  = mimtm2_l1.join(mimtm2_l2)
+  mimtm2_l.output('MIMTM.2', "MIMTM.2 : Minimum MiM bottom plate overlap of Vian-1 layer.
+                   [This is applicable for Vian-1 within 1.06um oversize of FuseTop layer
+                   (referenced to virtual bottom plate)]: 0.4µm")
+  mimtm2_l1.forget
+  mimtm2_l2.forget
+  mimtm2_l.forget
+  mimtm_via.forget
+
+  # Rule MIMTM.3: Minimum MiM bottom plate overlap of Top plate is 0.6um.
+  logger.info('Executing rule MIMTM.3')
+  mimtm3_l1 = fusetop.enclosed(mimtm_virtual, 0.6.um).polygons(0.001.um)
+  mimtm3_l2 = fusetop.not_inside(mimtm_virtual)
+  mimtm3_l = mimtm3_l1.join(mimtm3_l2)
+  mimtm3_l.output('MIMTM.3', 'MIMTM.3 : Minimum MiM bottom plate overlap of Top plate: 0.6um')
+  mimtm3_l.forget
+  mimtm3_l1.forget
+  mimtm3_l2.forget
+  mimtm_virtual.forget
+
+  # Rule MIMTM.4: Minimum MiM top plate (FuseTop) overlap of Vian-1 is 0.4µm.
+  logger.info('Executing rule MIMTM.4')
+  mimtm4_l1 = fusetop.enclosing(top_via, 0.4.um, euclidian).polygons(0.001.um)
+  mimtm4_l2 = top_via.not_outside(fusetop).not(fusetop)
+  mimtm4_l  = mimtm4_l1.join(mimtm4_l2)
+  mimtm4_l.output('MIMTM.4', 'MIMTM.4 : Minimum MiM top plate (FuseTop) overlap of Vian-1: 0.4µm')
+  mimtm4_l1.forget
+  mimtm4_l2.forget
+  mimtm4_l.forget
+
+  # Rule MIMTM.5: Minimum spacing between top plate and the Vian-1 connecting to the bottom plate is 0.4µm.
+  logger.info('Executing rule MIMTM.5')
+  mimtm5_l1 = fusetop.separation(top_via.interacting(topmin1_metal), 0.4.um, euclidian)
+  mimtm5_l1.output('MIMTM.5',
+                   'MIMTM.5 : Minimum spacing between top plate and the Vian-1 connecting to the bottom plate: 0.4µm')
+  mimtm5_l1.forget
+
+  # Rule MIMTM.6: Minimum spacing between unrelated top plates is 0.6µm.
+  logger.info('Executing rule MIMTM.6')
+  mimtm6_l1 = fusetop.space(0.6.um, euclidian)
+  mimtm6_l1.output('MIMTM.6', 'MIMTM.6 : Minimum spacing between unrelated top plates: 0.6µm')
+  mimtm6_l1.forget
+
+  # Rule MIMTM.7: Min FuseTop enclosure by CAP_MK is 0um.
```

Comment:
does that effectively means that all fusetop should be enclosed in capmk?

Reply:
@atorkmabrains Yes, exactly